static/app/views/app/index.tsx

```diff
@@ -170,6 +172,18 @@ function App({children, params}: Props) {
       );
     }
 
+    if (partnershipAgreementPrompt) {
+      return (
+        <Suspense fallback={null}>
+          <PartnershipAgreement
+            partnerDisplayName={partnershipAgreementPrompt.partnerDisplayName}
```

Comment:
Can we simply pass the `partnershipAgreementPrompt` and deconstruct it within the component?

Reply:
It's during the app boot and gets called with every url load 😅 so I didn't want to do anything out of the patterns they already have.